extension/WebJobs.Extensions.RabbitMQ/Trigger/RabbitMQListener.cs

```diff
@@ -82,6 +86,8 @@ public Task StartAsync(CancellationToken cancellationToken)
 
             this.consumer.Received += async (model, ea) =>
             {
+                Activity activity = StartActivity(ea); // create activity object
+
                 var input = new TriggeredFunctionData() { TriggerValue = ea };
```

Comment:
Can you use the using statement on Activity object? The `Dispose` method will automatically [call `Stop` on the Activity](https://github.com/dotnet/runtime/blob/55e2378d86841ec766ee21d5e504d7724c39b53b/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs#L1009-L1012). The using statement can [handle `null` Activity objects]([https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/statements#the-using-statement:~:text=If%20a%20null%20resource%20is%20acquired%2C%20then%20no%20call%20to%20Dispose%20is%20made%2C%20and%20no%20exception%20is%20thrown](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/statements#the-using-statement:~:text=If%20a%20null%20resource%20is%20acquired%2C%20then%20no%20call%20to%20Dispose%20is%20made%2C%20and%20no%20exception%20is%20thrown).).